lib/streamlit/elements/layouts.py

```diff
@@ -820,9 +830,11 @@ def popover(
         if label is None:
             raise StreamlitAPIException("A label is required for a popover")
 
+        if use_container_width is not None:
+            width = "stretch" if use_container_width else "content"
+
         popover_proto = BlockProto.Popover()
         popover_proto.label = label
-        popover_proto.use_container_width = use_container_width
```

Comment:
nit: I think we can mark the `use_container_width` proto property as deprecated.